smithy-model/src/test/java/software/amazon/smithy/model/validation/suppressions/MetadataSuppressionTest.java

```diff
@@ -0,0 +1,64 @@
+package software.amazon.smithy.model.validation.suppressions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import software.amazon.smithy.model.node.Node;
+import software.amazon.smithy.model.node.ObjectNode;
+import software.amazon.smithy.model.shapes.Shape;
+import software.amazon.smithy.model.shapes.StringShape;
+import software.amazon.smithy.model.validation.Severity;
+import software.amazon.smithy.model.validation.ValidationEvent;
+
+public class MetadataSuppressionTest {
+
+    @Test
+    public void doesNotMatchWhenUsingDifferentNamespace() {
+        ObjectNode node = Node.objectNodeBuilder()
+                .withMember("id", "Foo")
+                .withMember("namespace", "smithy.example.nested")
+                .build();
+        Shape s = StringShape.builder().id("smithy.example#String").build();
```

Comment:
Doesn't look like it's used, which means this test probably isn't doing a namespace comparison.